specification/sql/resource-manager/Microsoft.Sql/preview/2024-08-01-preview/DatabaseAdvisors.json

```diff
@@ -250,6 +285,157 @@
         }
       }
     },
+    "RecommendedAction": {
+      "description": "Database, Server or Elastic Pool Recommended Action.",
+      "type": "object",
+      "properties": {
+        "kind": {
+          "description": "Resource kind.",
+          "type": "string",
+          "readOnly": true
+        },
+        "location": {
+          "description": "Resource location.",
+          "type": "string",
+          "readOnly": true
+        },
+        "properties": {
+          "$ref": "#/definitions/RecommendedActionProperties",
+          "description": "Resource properties.",
+          "x-ms-client-flatten": true
+        },
+        "id": {
+          "description": "Resource ID.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "Resource name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "Resource type.",
+          "type": "string",
+          "readOnly": true
+        }
```

Comment:
it was referring to common-types ProxyResource. Why to remove now as that is the suggested way